pkg/linear-pools/contracts/silo-linear-pool/SiloExchangeRateModel.sol

```diff
@@ -30,19 +30,19 @@ contract SiloExchangeRateModel {
     using FixedPoint for uint256;
```

Comment:
The linter won't accept hints placed in the middle of multi-line comments, so let's put this link on its own line up here.

Also, I changed the link to point to the full state of the file at a given commit rather than a diff representing that commit. Our intention here is to analyze the behavior of the function at a point in time, whereas the diff view is better if we want to examine a specific change to the function.
```suggestion
    // Silo's `_accrueInterest` function is the basis for this contract's up-to-date exchange rate computation:
    // https://github.com/silo-finance/silo-core-v1/blob/70621494b23d333e67442485d7c743ee6e7d22db/contracts/BaseSilo.sol#L666
    // solhint-disable-previous-line max-line-length

    using FixedPoint for uint256;
```